src/application/environment.rs

```diff
@@ -0,0 +1,71 @@
+
+#[derive(Debug, Clone)]
+pub struct EffiEnvironment {
+    pub todo_directory_path: String,
+    pub todo_file_path: String,
+    pub done_file_path: String
+}
+
+fn initialize_effi_file(file_path: &str){
+    // If a todo.txt or done.txt file does not exist, create an empty one
+    if !std::fs::metadata(file_path).is_ok(){
+        log::info!("todo.txt or done.txt does not exist, creating an empty file at {}", file_path);
```

Comment:
The log messages are displaying in user interface in a small popup, this message is too long and probably should be a debug message.

Reply:
Swapped this to log as a debug message